app/Services/MessageBroker.php

```diff
@@ -23,11 +23,10 @@ public function publishRaw($payload, $routingKey)
         // Configure the message broker connection
         $credentials = config('services.message_broker.credentials');
         $config = config('services.message_broker.config');
-        $config['routingKey'] = $routingKey;
         $broker = new MessageBrokerConnection($credentials, $config);
 
         $serializedPayload = serialize($payload);
-        $broker->publishMessage($serializedPayload);
+        $broker->publish($serializedPayload, $routingKey);
```

Comment:
This is set [from an environment variable](https://github.com/DFurnes/voting-app/blob/c124986f71adb9794fdef8c737381309450e718b/.env.example#L25) so it can be environment-specific (e.g. `vote.event.CGG` here, but `vote.event.AGG` on the AGG site).